meta-resin-common/recipes-support/resin-mounts/resin-mounts/mnt-conf-systemd.automount

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Automount systemd presistent state
+
+[Automount]
```

Comment:
This avoids the problem of ordering. This way the services that need that directory don't need to be explicitly listed.